scripts/integration-tests/patches/vue-cli-babel-8.patch

```diff
@@ -0,0 +1,128 @@
+diff --git a/packages/@vue/babel-preset-app/__tests__/babel-preset.spec.js b/packages/@vue/babel-preset-app/__tests__/babel-preset.spec.js
+index d57f42b..724bb37 100644
+--- a/packages/@vue/babel-preset-app/__tests__/babel-preset.spec.js
++++ b/packages/@vue/babel-preset-app/__tests__/babel-preset.spec.js
+@@ -3,7 +3,8 @@ const babel = require('@babel/core')
+ const preset = require('../index')
+ const defaultOptions = {
+   babelrc: false,
+-  presets: [preset],
++  targets: { ie: 9 },
++  presets: [[preset, { targets: { ie: 9 } }]],
+   filename: 'test-entry-file.js'
+ }
+ 
+@@ -21,6 +22,7 @@ test('polyfill detection', () => {
+     const a = new Map()
+   `.trim(), {
+     babelrc: false,
++    targets: { ie: 9 },
```

Comment:
Q: Why should we patch given that the preset has specified `{ ie: 9 }` in targets option?

Reply:
Oh right, it's not necessary. It was my first attempt at fixing these tests, and since it reduced the number of failure I didn't realize it's not what caused them.